src/common/fs/write-json-sync.injectable.ts

```diff
@@ -3,11 +3,29 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 import { getInjectable } from "@ogre-tools/injectable";
+import getDirnameOfPathInjectable from "../path/get-dirname.injectable";
 import fsInjectable from "./fs.injectable";
 
+export type WriteJsonSync = (filePath: string, contents: unknown) => void;
+
 const writeJsonSyncInjectable = getInjectable({
   id: "write-json-sync",
-  instantiate: (di) => di.inject(fsInjectable).writeJsonSync,
+  instantiate: (di): WriteJsonSync => {
+    const {
+      writeJsonSync,
+      ensureDirSync,
+    } = di.inject(fsInjectable);
+    const getDirnameOfPath = di.inject(getDirnameOfPathInjectable);
+
+    return (filePath, content) => {
+      ensureDirSync(getDirnameOfPath(filePath), { mode: 0o755 });
+
+      writeJsonSync(filePath, content, {
+        encoding: "utf-8",
+        spaces: 2,
```

Comment:
These could be always set in `writeJsonSync`.

Reply:
What do you mean? This is `wrieJsonSync`